From 1b97a5d2c0413fdcc7cd3a9f541b7aa88ed74a39 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sat, 2 Mar 2024 15:13:04 +0100
Subject: [PATCH 419/463] arm64: dts: rk3588-orangepi-5-plus: Add dwc3 reset
 quirk

This is needed to apply type-c muxing/orientation properly to the phy.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
index 3245b2c1a259b..38ec2a1183fda 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-orangepi-5-plus.dts
@@ -924,6 +924,7 @@ &usb_host0_ohci {
 
 &usb_host0_xhci {
 	usb-role-switch;
+	snps,usb3-phy-reset-quirk;
 	status = "okay";
 
 	port {
-- 
2.46.0

